dotcom-rendering/src/model/enhanceCards.ts

```diff
@@ -182,6 +184,32 @@ const enhanceTags = (tags: FETagType[]): TagType[] => {
 	});
 };
 
+const extractVideo = (
+	mediaAtom?: FEMediaAtom,
+): CardYoutubeVideo | undefined => {
+	if (!mediaAtom) return undefined;
+	const asset = mediaAtom.assets.find(
+		({ version }) => version === mediaAtom.activeVersion,
+	);
+	if (asset?.platform === 'Youtube') {
+		return {
+			elementId: mediaAtom.id,
+			videoId: asset.id,
+			// Size fixed to a 5:3 ratio
+			width: 500,
+			height: 300,
```

Comment:
Cards do not have a concept of an intrinsic size, and calculating the size of the poster image seemed brittle.